docs: add CVE-2022-29266 post (#1051)
iresty/iresty.github.io: website/blog/2022/04/20/cve-2022-29266.md

---
title: "The Vulnerability of Leaking Information in Error Response from jwt-auth Plugin（CVE-2022-29266）"
keywords: 
- Vulnerability
- jwt-auth
- Error Response

> description: In APISIX 2.13.1 and previous versions, there is a problem of information leakage caused by the `jwt- auth` plug-in.

Reviewer: ```suggestion
description: In APISIX 2.13.0 and previous versions, there is a problem of information leakage caused by the `jwt- auth` plug-in.
```

tags: [Security]
---

> > In APISIX 2.13.1 and previous versions, there is a problem of information leakage caused by the `jwt- auth` plug-in.

Reviewer: ```suggestion
> In APISIX 2.13.0 and previous versions, there is a problem of information leakage caused by the `jwt- auth` plug-in.
```

<!--truncate-->

## Problem Description

> The `jwt- auth` plug-in has a security problem of leaking the user's secret key because the error message returned from the dependent library lua-resty-jwt contains sensitive information.

Reviewer: ```suggestion
The `jwt- auth` Plugin has a security problem of leaking the user's secret key because the error message returned from the dependent library `lua-resty-jwt` contains sensitive information.
```
